src/test/rustdoc/const-evalutation-ice.rs

```diff
@@ -17,4 +17,4 @@ pub struct S {
     s: Cell<usize>
 }
 
-pub type _S = [usize; 0 - (mem::size_of::<S>() != 4) as usize];
+pub const N: usize = 0 - (mem::size_of::<S>() != 4) as usize;
```

Comment:
Hu, why did this change? Now the comment in line 11 makes no sense any more.

Reply:
I'm not sure of @oli-obk's exact intention here, but I've resolved it in the way that seems most obvious.

Reply:
Uh, previously this referred to the `type _S` that oli removed, now you made it refer to the `struct S` instead. That seems like a change in content to me.

Reply:
I think what happened is that `_S` was a bogus type so rustdoc ICEd. That got fixed in https://github.com/rust-lang/rust/commit/6b35d813826a8011e3b3f19206c22f6978173134. Now the code actual errors before rustdoc can even start. But we can at least still add a bogus constant and see if rustdoc works on that.

So, I think the comment should refer to `N`, not `S`.

Reply:
Okay, but `N` isn't a type, so I really wasn't sure what Oli meant...